documentation/api/parameters.md

```diff
@@ -123,23 +111,62 @@ Authorization: Bearer fffffffffffffffffffffffffffffffffffffffffff=
 ### Sample Response
 
 ```http
-HTTP/1.1 202 Accepted
+HTTP/1.1 200 OK
 Content-Type: application/json
```

Comment:
This sample is no longer valid, correct? It doesn't return `application/json` anymore and there is not `captures` property on the root object that I can find.

Reply:
Oops, I forgot to update.